lms/models/region.py

```diff
@@ -1,7 +1,7 @@
 from dataclasses import dataclass
 
 
-@dataclass
+@dataclass(frozen=True)
```

Comment:
Might as well put the `frozen=True` back now that I can. It's not needed, but it's potentially informative to know that regions are never mutated